msteams-platform/TOC.yml

```diff
@@ -104,6 +104,8 @@
     href: concepts/build-and-test/teams-developer-portal.md
   - name: Teams JavaScript client SDK
     href: tabs/how-to/using-teams-client-sdk.md
+  - name: Teams JavaScript client SDK v2 Preview
```

Comment:
Global comment. Please update the Developer preview section in the What's new file:

https://docs.microsoft.com/en-us/microsoftteams/platform/whats-new#developer-preview

Reply:
Added in next commit

Reply:
How does this look: https://review.docs.microsoft.com/en-us/microsoftteams/platform/whats-new?branch=pr-en-us-4554#developer-preview